OpTestConfiguration.py

```diff
@@ -385,6 +385,17 @@ def get_parser():
     kernelcmdgroup.add_argument("--remove-kernel-args",
                                 help="Kernel commandline option to be removed",
                                 default="")
+    networkdumpgroup = parser.add_argument_group("server credentials for network dumps",
+                                               "ipaddress username and password of remote server")
+    networkdumpgroup.add_argument("--server-ip",
+                                help="IP address of remote server where dump will be stored",
+                                default="")
+    networkdumpgroup.add_argument("--server-pw",
+                                help="Passw0rd of remote server where dump will be stored",
+                                default="")
+    networkdumpgroup.add_argument("--net-path",
+                                help="Directory path of remote server where dump will be stored",
+                                default="/var/crash")
     cronusgroup = parser.add_argument_group("Cronus", "Cronus Config options")
```

Comment:
@Pavithra1602 I see only addition of commaline parameters for kdump. The intension is to use it in future? 

can we have dump prefix to all the argument ex dump-server-ip, dump-server-password, dump_net_path. 

- Hariharan

Reply:
Just adding new argument without using it is not useful Also patch doesn't describe how you are going to use it.

Also may be we should create separate kdump group?

-Vasant